QuizAppNew/View/QuestionView.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  QuestionView.swift
+//  QuizAppNew
+//
+//  Created by Jannatun Nahar Papia on 21/8/22.
+//
+
+import SwiftUI
+
+struct QuestionView: View {
+    
+    @Binding var question: Qustion
+    @Binding var correct: Int
+    @Binding var wrong: Int
+    @Binding var answered: Int
+    @State var selected = ""
+    
+    var body: some View {
+        VStack(spacing: 22){
+            // FOR QUESTION....
+            Text(question.question) // Its right
+                .font(.title2)
+                .fontWeight(.heavy)
+                .foregroundColor(Color.black)
+                .padding(.top, 25)
+            
+            Spacer(minLength: 0)
+            
+            // Buttons
+        //    Button(action: {selected = question.optionA}, label: { // right
+            Button(action: {}, label: {
+                Text(question.optionA)
```

Comment:
are you stupid? I told you only UI design why use model with property gets option text. use dummy text.

Reply:
UI design is Added